api.raml

```diff
@@ -0,0 +1,107 @@
+#%RAML 1.0
+title: ID Broker API
+protocols: [ HTTPS ]
```

Comment:
Is there a way we can design it to always use https (even behind the load balancer)? Since we're passing around passwords in plaintext, always using https (even on a private network) seems like a good practice, especially if it's easy.